### DISCUSSION TITLE
Phase 1 question

### DISCUSSION THREAD

**Discussion by leonidcan** [ericaltendorf/plotman#634] [Q&A]
Why most of the time plots are in sleep state in phase one? Do I do anything wrong?

![image](https://user-images.githubusercontent.com/18264548/119244307-48942680-bb3d-11eb-9d5c-fe7eb6e01245.png)

**Comment by leonidcan** [comment]
Anyone can answer my question? It seems that I can optimize more with my setup (also I often have much less jobs in phase 2 than 1 or 3).

**Reply by pspardon** [reply]
This is normal, check the [plot statuses wiki page.](https://github.com/ericaltendorf/plotman/wiki/Plot-statuses)
